### ISSUE TITLE
cSpell: Ignore the core/modules/ckeditor5/lib/ckeditor5/build directory

### ISSUE THREAD

**Issue by kiamlaluno** [backdrop/backdrop-issues#6521]
The _core/modules/ckeditor5/lib/ckeditor5/build_ directory contains files generated by core/modules/ckeditor5/lib/scripts/concat-build.sh.  
cSpell should ignore that directory, since those files are not supposed to be manually edited. If there is anything that needs to be fixed, that needs to be done on the source files.

**Comment by klonos** [comment]
Thank you @kiamlaluno 🙏🏼 ...that was in my to-do list, so it was pleasant to see that not only you have opened an issue for it, but also provided a PR. RTBC!

Trying to draw the attention of our @backdrop/core-committers to this simple one-liner that will reduce noise in our PR checks.

**Comment by laryn** [comment]
Thank you @kiamlaluno for the report and the PR, and for testing it out @klonos. Makes sense and looks good. I saw the milestone was `1.27.2` but I don't think this is relevant on anything but `1.x` so I've just merged it there.

**Comment by klonos** [comment]
> ... I saw the milestone was 1.27.2 but I don't think this is relevant on anything but `1.x` so I've just merged it there.

Yeah, I was thinking that CKE5 is already in 1.27.x, but you are right: our PRs are always against 1.x 👍🏼
